src/Command/Anonymization/AnonymizeCommand.php

```diff
@@ -355,27 +332,16 @@ private function doBackupAnonymizedDatabase(): void
         }
 
         $backupper = $this->backupperFactory->create($this->connectionName);
-        // If we are not anomymizing a database from a given backup file, we put
-        // anonymized database backup in classic storage dir but we specify
-        // it's anonymized.
-        $this->backupFilename = $this->backupFilename ?? $this->storage->generateFilename($this->connectionName, $backupper->getExtension(), true);
-        $backupper
-            ->setDestination($this->backupFilename)
-            ->setVerbose($this->io->isVerbose())
-            ->startBackup()
-        ;
-
-        foreach ($backupper as $data) {
-            if ($this->io->isVerbose()) {
-                $this->io->text($data);
-            }
-        }
-
-        $backupper->checkSuccessful();
         if ($this->io->isVerbose()) {
-            $this->io->text($backupper->getOutput());
+            $backupper->setVerbose(true)->addLogger(new ConsoleLogger($this->io));
```

Comment:
Don't use a setVerbose() option on backupers, simply always log, but using level. The logger configuration will or will not display items depending on their level. That's useless logic IMHO.

Reply:
I just kept the initial logic as it was, but it's actually no longer relevant now that we use loggers. I will change that.

Reply:
In fact, `setVerbose()` is not initially used to control the logging verbosity of the backupper itself, it is mainly to tell the backupper to call the binary in verbose mode. I will keep it, but I will add the logger independently of the verbosity level, `ConsoleLogger` handles verbosity by itself.